.github/workflows/build.yml

```diff
@@ -6,10 +6,11 @@ name: Build
 on:
   push:
     branches:
-      - '*'
+      - '**'
```

Comment:
Updates to ensure we push whenever any branch is pushed, this was a bug from the initial GHA rollout.